index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html xml:lang="en" lang="en">
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  <title>Portfolio Project</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;700;800&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+  <link rel="stylesheet" type="text/css" href="style.css" />
+</head>
+
+<body>
+
+  <div class="header">
+    <img class="logo" src="My Logo.png">
```

Comment:
- [ ] Usually, the logo is a clickable area so I highly recommend wrapping this line inside an anchor element `<a>`  👍 and please your personal logo, or simply your name.

eg:
```html
<a class="logo">your_name</a>
```